src/ctia/stores/es/sort.clj

```diff
@@ -18,6 +27,35 @@
       ;; https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_sort_values
       {field-name {:order order}}
 
+      ;; chooses the maximum value in field (a list) based on remappings,
+      ;; and sorts based on that value for each entity. Remappings must
+      ;; be positive integers.
+      ;; eg
+      ;; {:op :remap-list-max
+      ;;  :field-name "tactics"
+      ;;  :remappings {"TA0043" 1
+      ;;               "TA0042" 2}
+      ;;  :sort_order :asc}
+      :remap-list-max
+      (let [{:keys [remap-default remappings]} params
+            remappings (normalize-remappings remappings)]
+        (assert (every? (complement neg?) (vals remappings)) ":remap-list-max :remappings values must be non-negative")
```

Comment:
I guess `remappings` is set by the developers, but if this `remappings` is constructed somehow, the assertion could fail in a deployed env, and not in the developer env.

So perhaps would it be safer to replace this `assert` by throwing an Exception with detailed logs, but not much detail in the Exception itself, so the HTTP error could return a 500 without much detail about the issue. 

I'm not sure the middleware already handle correctly or not these kind of exception. So I could be totally wrong here.

Reply:
I removed the assert since it didn't make much sense anyway (it reflected a previous iteration).